frontend/src/app/profile/profile-editor/profile-editor.component.html

```diff
@@ -92,3 +92,25 @@
     </mat-card-actions>
   </ng-template>
 </mat-card>
+
+<html>
+  <mat-card appearance="outlined">
+    <mat-card-actions class="token">
+      <button mat-stroked-button type="submit" (click)="displayToken()">
+        Display Bearer Token
+      </button>
+      <button
+        class="cancel-button"
+        mat-stroked-button
+        type="submit"
+        (click)="copyToken()">
+        Copy
+      </button>
+    </mat-card-actions>
+    <div *ngIf="token">
```

Comment:
I may take a stab at a slight refactoring here to make this not dependent on the token such that the token can be loaded in construction.